README.md

```diff
@@ -1,2 +1,29 @@
-# bookstore
-Book Store app made using react and redux
+# Bookstore
+Book Store app made using react and redux. It will help you organize and manage bookstore.
+
```

Comment:
**I trust you can update your `README` file with this `https://bookstore-cms-demo.herokuapp.com/` before merging.
A `README` file without live link doesn't look professional.**

Reply:
Sure We will update it. Thank you